README.md

```diff
@@ -70,82 +116,135 @@ Docker is needed to run the project in a containerized environment. Follow the a
 Ensure `docker-compose` is installed. Refer to the [Docker Compose installation guide](https://docs.docker.com/compose/install/).
 
 ### Setting Up Environment Variables
-Create or update the `.env` files in the `fastapi_backend` and `nextjs-frontend` directories with the following variables:
 
 **Backend (`fastapi_backend/.env`):**
-
-For local setup:
-```
-DATABASE_URL=postgresql+asyncpg://postgres:password@localhost:5432/mydatabase
-```
-For Docker setup, replace localhost with db:
-```
-DATABASE_URL=postgresql+asyncpg://postgres:password@db:5432/mydatabase
-```
-Other environment variables remain unchanged:
-
-```env
-ACCESS_SECRET_KEY=<your-secret-key>
-RESET_PASSWORD_SECRET_KEY=<your-secret-key>
-VERIFICATION_SECRET_KEY=<your-secret-key>
-OPENAPI_OUTPUT_FILE=../nextjs-frontend/openapi.json
-
-MAIL_USERNAME=<your-username>
-MAIL_PASSWORD=<your-password>
-MAIL_FROM=<your-from>
-MAIL_SERVER=<your-server>
+Copy the `.env.example` files to `.env` and update the variables with your own values.
+```bash
+cp project_name/.env.example project_name/.env
 ```
 
 **Frontend (`nextjs-frontend/.env.local`):**
-```env
-NEXT_PUBLIC_API_BASE_URL=http://localhost:8000
-OPENAPI_OUTPUT_FILE=openapi.json
+Copy the `.env.local.example` files to `.env.local` and update the variables with your own values.
+```bash
+cp project_name/.env.local.example project_name/.env.local
 ```
 
-### If you are not using Docker:
-To run the project locally, use the following commands:
-
-#### Backend
-1. Navigate to the `fastapi_backend` directory.
-2. Use Docker to run the database to avoid local installation issues. Build and start the database container:
+### Running the Database
+1. Use Docker to run the database to avoid local installation issues. Build and start the database container:
    ```bash
    docker compose build db
    docker compose up db
    ```
-3. Run the following command to apply database migrations:
+2. Run the following command to apply database migrations:
    ```bash
    make docker-migrate-db
    ```
-4. Start the FastAPI server:
+
+### If you are not using Docker:
```

Comment:
Change to: "Run the project (without Docker)"
and the other section should be: "Run the project (with Docker)"